### DISCUSSION TITLE
Erratic UI behavior on hover Q-Select, Q-Button, Q-Tabs

### DISCUSSION THREAD

**Discussion by carlgwsmith** [quasarframework/quasar#12028] [General - Components / Directives / etc]
I have been using Quasar v1 on a development environment with google chrome v97 on mac osx and am getting very erratic behavior that seems to be randomly triggered when q-selects, q-buttons, q-tabs, q-inputs and more are clicked or hovered on. Has anyone seen this type of behavior and is there a remedy for it? 

Video:  https://streamable.com/ndj3du

Thanks,
Carl

**Comment by hawkeye64** [comment]
Definitely not Quasar. Something to do with your system, possibly a display driver.

**Reply by carlgwsmith** [reply]
I don't think it's a display issue, I am able to replicate the issue on Chrome in windows on several machines. Does not occur in firefox on any OS.

**Reply by hawkeye64** [reply]
Look at the video. Monitors draw things on their display in squares (from video memory). For a line to break like that, it's a display driver telling the monitor where to draw. The square segments are not lining up.
Have you tried on a different monitor and a different computer?
The only other thing I can think of is if your monitor has a pinned connection then check if one of the pins is missing or bent. If it is an HDMI connection, make sure it is plugged in all the way.

**Reply by carlgwsmith** [reply]
I have tried this on several machines. The application works well and without this issue in firefox. Chrome the issue is rampant.

**Reply by hawkeye64** [reply]
On Chrome, do you get the same behavior when going to other sites? What about quasar.dev? Is this in a dev environment or production? What extension for Chrome do you have running? Have you tried in incognito mode?

Do you have your site available where I can do some testing?
